tests/literature/test_docinsight_client.py

```
@@ -0,0 +1,78 @@
+    ]
+    def fake_post(url, json):
+        assert url == f"{base}/get_results"
+        assert json == {"job_ids": ["job-123"]}
+        return DummyResponse(200, expected)
+    monkeypatch.setattr(requests, "post", fake_post)
+    results = client.get_results(["job-123"])
+    assert results == expected
+
+### New tests for wait_for_result polling behavior ###
+import pytest
+from cultivation.scripts.literature.docinsight_client import DocInsightClient, DocInsightAPIError, DocInsightTimeoutError
```

Ruff (0.11.9)

Redefinition of unused `DocInsightClient` from line 2

Remove definition: `DocInsightClient`

(F811)